packages/idealo.design/src/ui/pages/BlogPage/DetailView.jsx

```diff
@@ -138,8 +155,16 @@ const BlogDetailView = (props) => {
                                   <span>Next</span>
                                 </Link>)}
       </div>
+
+      <Prompt
+          show = {prompt.isPromptOpen}
```

Comment:
Please remove the spaces around the `=` to keep the formatting the same as everywhere else.